partiql-eval/src/eval.rs

```diff
@@ -1018,6 +1070,7 @@ impl EvalExpr for EvalBinOpExpr {
     }
 }
 
+/// Represents an evaluation PartiQL `BETWEEN` operator, e.g. in `x BETWEEN 10 AND 20`.
```

Comment:
This expr models the whole between expression (i.e. `x BETWEEN 10 AND 20`)

```suggestion
/// Represents an evaluation PartiQL `BETWEEN` operator, e.g. `x BETWEEN 10 AND 20`.
```